src/test/scala/org/apache/spark/shuffle/TestUtil.scala

```diff
@@ -47,4 +47,15 @@ object TestUtil {
     conf.set("spark.shuffle.rss.serviceRegistry.zookeeper.servers", zooKeeperServers)
     conf
   }
+
+  def newSparkConfWithSequenceServerRegistryServer(appId: String): SparkConf ={
+    val conf = TestUtil.newSparkConfWithStandAloneRegistryServer(appId, "")
+    var testStreamServer: TestStreamServer =
```

Comment:
it shall be immutable. maybe change `var` to `val`
```suggestion
    val testStreamServer: TestStreamServer =
```

Reply:
> it shall be immutable. maybe change `var` to `val`

Done